ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+
+# Fix UID issue (from Telco QE Team)
+~/fix_uid.sh
+
+bastion=$(cat "/secret/address")
+
+cat <<EOF >>/tmp/all.yml
+---
+lab: scalelab
+lab_cloud:
+cluster_type: $TYPE
+worker_node_count: $NUM_WORKER_NODES
+sno_node_count:
+public_vlan: false
+ocp_release_image: $OCP_RELEASE_IMAGE
+openshift_version: "$OCP_VERSION_SHORT"
+networktype: OVNKubernetes
+enable_fips: $FIPS
+ssh_private_key_file: ~/.ssh/id_rsa
+ssh_public_key_file: ~/.ssh/id_rsa.pub
+pull_secret: "{{ lookup('file', '../pull_secret.txt') }}"
+bastion_cluster_config_dir: /root/{{ cluster_type }}
+smcipmitool_url:
+bastion_lab_interface: eno12399
+bastion_controlplane_interface: ens6f0
+controlplane_network: 192.168.216.1/21
+controlplane_network_prefix: 21
+bastion_vlaned_interface: ens1f1
+setup_bastion_gogs: false
+setup_bastion_registry: false
+use_bastion_registry: false
+controlplane_lab_interface: eno1np0
+controlplane_pub_network_cidr:
+controlplane_pub_network_gateway:
+jumbo_mtu: false
+rwn_lab_interface: eno1np0
+rwn_network_interface: ens1f0
+EOF
+
+envsubst < /tmp/all.yml > /tmp/all-updated.yml
+
+sshpass -p "$(cat /secret/login)" scp -q -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null /tmp/all-updated.yml root@${bastion}:~/jetlag/ansible/vars/all.yml
+sshpass -p "$(cat /secret/login)" scp -q -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null /secret/inventory root@${bastion}:~/jetlag/ansible/inventory/telco.inv
+
+# Clean up previous attempts
```

Comment:
For clarity, I would suggest:
```
export run_in_bastion='sshpass -p "$(cat /secret/login)" ssh -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null root@${bastion}'
$run_in_bastion echo command
$run_in_bastion bash <<EOF
echo first command
echo second command
hostname
EOF
```

Same for the copy, a `copy_to_bastion` could be defined.

Reply:
@josecastillolema it is already merged, if you want to do a follow on to clarify things